server.js

```diff
@@ -16,12 +17,77 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+const ThoughtSchema = new mongoose.Schema({
+	message: {
+		type: String,
+		required: true,
+		unique: true,
+		minlength: [5, 'The message must be at least 5 characters'],
+		maxlength: [140, 'The message must be less than 140 characters'],
```

Comment:
Ooh very cool! I didn't know we could add a message after the min or maxlength. Awesome!